src/utils/deployedTranslations.ts

```diff
@@ -0,0 +1,15 @@
+// This is a list of languages with enough translated content.
+// Add more languages here when they have enough content.
+// Please DO NOT edit this list without a discussion in the reactjs/react.dev repo.
+// It must be the same between all translations.
+// This will also affect the 'Translations' article.
+
+// prettier-ignore
+export const deployedTranslations = [
```

Comment:
Let's call this `finishedTranslations` since most/all of the in-progress translations are deployed.

Reply:
Actually, maybe we should upstream this list to `reactjs/translations.react.dev/main/langs/langs.json` so all translation sites get the update automatically.

It could be just a flag:

```
{ "code": "ar", "name": "العربية", "enName": "Arabic", finished:true},
{ "code": "az", "name": "Azərbaycanca", "enName": "Azerbaijani" },
```

Or even better, the current progress:

```
{ "code": "ar", "name": "العربية", "enName": "Arabic", progress: 99},
{ "code": "az", "name": "Azərbaycanca", "enName": "Azerbaijani", progress: 45},
```

The advantage of `progress` is that we can compute it dynamically so we don't need a PR to mark a language as "finished".

Reply:
Currently, there's no robust way to calculate the progress of each fork. https://translations.react.dev/ works by issuing the following GraphQL query _from the browser_ and count the number of checked/unchecked checkboxes (`[x]`/`[ ]`) on issues like [this](https://github.com/reactjs/ja.react.dev/issues/452).

```graphql
query($limit: Int!) {
  search(
    type: ISSUE
    query: "org:reactjs Translation Progress in:title is:open"
    first: $limit
  ) {
    nodes {
      ... on Issue {
        title
        body
        createdAt
        lastEditedAt
        number
        repository {
          name
        }
      }
    }
  }
}
```

Ideally, it's better to create a new cron GitHub Action in the translation repository to correctly calculate the progress and commit it as JSON, but this would be a nontrivial task.

Reply:
Ok we can ship with this for now, thanks for considering!